src/components/box_placeholder.tsx

```diff
@@ -2,21 +2,31 @@ import * as React from 'react';
 
 import BoxIcon from './icons/box_icon';
 
+export type BoxPlaceholderBackground = 'grey' | 'transparent';
```

Comment:
We have discussed moving colors into the realm of css, so 'grey' will be changed here to 'light', and wherever else it is needed.